admin/src/Components/Textarea.jsx

```
@@ -1,86 +1,82 @@
-        );
+    const textarea = (
+        <textarea
+            id={name}
+            name={name}
+            placeholder={placeholder}
+            className="uk-width-1-1"
+            value={value}
+            disabled={disabled}
+            tabIndex={tabIndex}
+            rows={rows || 8}
+            onChange={(e) => (model[name] = e.target.value)}
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)